docs/tutorial_hist.rst

```diff
@@ -458,6 +458,29 @@ can find this class in the ``xonsh.teepty`` module. Like with lazy JSON, this is
 from other parts of xonsh and can be used on its own.  If you find this useful in other areas,
 please let us know!
 
+
+Sqlite backed History
+=====================
+
+Xonsh has a second built-in history backend powered by sqlite (other than
+the JSON version mentioned all above in this tutorial). It shares the same
+functionalities as the JSON version in the most ways, except it currently
+doesn't support ``history diff`` and ``history replay`` actions.
+
+When there are a lot of history JSON files, before it hitting the
```

Comment:
"...before hitting the ``history gc`` thresholds, it can take some time to load all of the history files when xonsh is launched. "